mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -200,6 +204,11 @@ class CrossValidatorModel private[ml] (
     @Since("1.5.0") val avgMetrics: Array[Double])
   extends Model[CrossValidatorModel] with CrossValidatorParams with MLWritable {
 
+  /** A Java/Python-friendly auxiliary constructor. */
+  private[ml] def this(uid: String, bestModel: Model[_], avgMetrics: ju.List[Double]) = {
```

Comment:
Similar comment to the one bellow, if this is intended for both Java and Python then we probably don't want it to be private ml scope - but it seems like the intended use of this is Python and this might be better suited to PythonMLLibAPI.scala